recipes/xvfbwrapper/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.2.8" %}
+
+package:
+  name: xvfbwrapper
+  version: {{ version }}
+
+source:
+  fn: xvfbwrapper-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/x/xvfbwrapper/xvfbwrapper-{{ version }}.tar.gz
+  md5: c8ce7c35fc555a6af272e7e17cf0b5c0
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
So normally we only need these arguments if this uses `setuptools`. Does this package require `setuptools`? If so, we should add it to `requirements/build`. If not, we can drop these.